embulk-docs/src/built-in.rst

```diff
@@ -353,13 +353,25 @@ The ``json`` parser plugin parses a JSON file that contains a sequence of JSON o
-+----------------------------+----------+----------------------------------------------------------------------------------------------------------------+------------------------------+
++----------------------------+----------+-----------------------------------------------------------------------------------------------------------------+----------------------------------------+
+| name                       | type     | description                                                                                                     |          required?                     |
++============================+==========+=================================================================================================================+========================================+
+| stop\_on\_invalid\_record  | boolean  | Stop bulk load transaction if a file includes invalid record (such as invalid json)                             | ``false`` by default                   |
++----------------------------+----------+-----------------------------------------------------------------------------------------------------------------+----------------------------------------+
+| invalid\_string\_escapes   | enum     | Escape strategy of invalid json string such as using invalid ``\`` like ``\a``. (PASSTHROUGH, SKIP, UNESCAPE)   | ``PASSTHROUGH`` by default             |
++----------------------------+----------+-----------------------------------------------------------------------------------------------------------------+----------------------------------------+
+| root                       | string   | Specify when pointing a JSON object value as a record via JSON pointer expression                               | optional                               |
++----------------------------+----------+-----------------------------------------------------------------------------------------------------------------+----------------------------------------+
+| flatten_json_array         | boolean  | Set true when regard elements in a JSON array as multiple Embulk records.                                       | ``false`` by default                   |
++----------------------------+----------+-----------------------------------------------------------------------------------------------------------------+----------------------------------------+
+| columns                    | hash     | Columns (see below)                                                                                             | optional                               |
++----------------------------+----------+-----------------------------------------------------------------------------------------------------------------+----------------------------------------+
+| default\_timezone          | string   | Time zone of timestamp columns. This can be overwritten for each column using ``column``                        | ``UTC`` by default                     |
```

Comment:
Yeah, `default_timezone`, `default_timestamp_format`, and `default_date` can be configured actually, but in a rough direction, we're going to deprecate them.

Sorry that I didn't let you know, but let's keep them undocumented.